web/src/components/njwds-layout/ActionBarLayout.tsx

```diff
@@ -5,6 +5,7 @@ interface Props {
   children: ReactNode;
   stackOnLeft?: ReactNode;
   alignLeft?: boolean;
+  dontReverseInMobile?: boolean;
```

Comment:
Can we have this be the opposite? `reverseInMobile`? The double negative can be a bit harder to read

Reply:
added